examples/air/viz.py

```diff
@@ -35,7 +35,7 @@ def draw_one(imgarr, z_arr):
     # Note that this clipping makes the visualisation somewhat
     # misleading, as it incorrectly suggests objects occlude one
     # another.
-    clipped = np.clip(imgarr.data.cpu().numpy(), 0, 1)
+    clipped = np.clip(imgarr.detach().cpu().numpy(), 0, 1)
```

Comment:
Is it safe to replace `.detach().cpu().numpy()` with `.cpu().numpy()` everywhere?

Reply:
That's what I started with. If the node has `requires_grad=True`, it will throw an exception when we convert it to numpy without detaching.